sssom/util.py

```diff
@@ -944,9 +944,10 @@ def curie_from_uri(uri: str, prefix_map: Mapping[str, str]) -> str:
 def get_prefixes_used_in_table(df: pd.DataFrame) -> List[str]:
     """Get a list of prefixes used in CURIEs in key feature columns in a dataframe."""
     prefixes = []
-    for col in KEY_FEATURES:
-        for v in df[col].values:
-            prefixes.append(get_prefix_from_curie(v))
+    if not df.empty:
```

Comment:
Why not use same len() logic as in the other case?

Reply:
because we don't need to check columns and rows individually for this particular case.